subprojects/signing/src/main/java/org/gradle/plugins/signing/Signature.java

```diff
@@ -193,6 +198,8 @@ private void init(Callable<File> toSign, Callable<String> classifier, Callable<S
      *
      * @return The file. May be {@code null} if unknown at this time.
      */
+    @PathSensitive(PathSensitivity.NAME_ONLY)
```

Comment:
Does the name really matter? It matters only for the output file name, but we already track that separately.

Reply:
I only added it because `Sign.getInputFiles()` was using it. IMHO it doesn't matter.

Reply:
Then let's use `PathSensitivity.NONE`.